common/test/acceptance/tests/studio/test_studio_container.py

```diff
@@ -1136,3 +1137,230 @@ def test_common_problem_types_tab(self):
+        Then I see moved component there
+        """
+        unit_page = self.go_to_unit_page(unit_name='Test Unit 1')
+        components = unit_page.displayed_children
+        self.assertEqual(len(components), 2)
+
+        self.verify_move_opertions(
+            unit_page=unit_page,
+            source_component=components[0],
+            operation='move',
+            component_display_names_after_operation=['HTML 21', 'HTML 22', 'HTML 11']
+        )
+
+    def test_undo_move(self):
+        """
+        Test if we can undo move a component successfully.
+
+        Given I am a staff user
+        When I go to unit page in first section
+        Then I open the move modal
+        Then I click on the move button
+        Then I see move operation successful message
+        When I clicked on undo move link
+        Then I verified that undo move operation is successful
+        """
+        unit_page = self.go_to_unit_page(unit_name='Test Unit 1')
+        components = unit_page.displayed_children
+        self.assertEqual(len(components), 2)
+
+        self.verify_move_opertions(
+            unit_page=unit_page,
+            source_component=components[0],
+            operation='undo_move',
+            component_display_names_after_operation=['HTML 11', 'HTML 12']
+        )
+
+    def test_content_experiment(self):
+        """
+        Test if we can move a component of content experiment successfully.
+
+        Given I am a staff user
+        When I go to content experiment page
+        Then I open the move modal
+        Then I navigate to unit in second section from within move modal
+        Then I see move button is enabled
+        Then I click on the move button
+        Then I see move operation successful message
+        When I go to unit page in second section
+        Then I see moved component there
+        When I undo move a component
+        Then I verified that undo move operation is successful
+        """
+        # Add content experiment support to course.
+        self.course_fixture.add_advanced_settings({
+            u'advanced_modules': {'value': ['split_test']},
+        })
+
+        # Create group configurations
+        self.course_fixture._update_xblock(self.course_fixture._course_location, {
+            'metadata': {
+                u'user_partitions': [
+                    create_user_partition_json(
+                        0,
+                        'Test Group Configuration',
+                        'Description of the group configuration.',
+                        [Group('0', 'Group A'), Group('1', 'Group B')]
+                    ),
+                ],
+            },
+        })
+
+        # Add split test to unit_page1 and assign newly created group configuration to it
+        split_test = XBlockFixtureDesc('split_test', 'Test Content Experiment', metadata={'user_partition_id': 0})
+        self.course_fixture.create_xblock(self.unit_page1.locator, split_test)
```

Comment:
We are only using `split_test` in this test, so I don't think we should add it to all other tests